sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/resources/templates/welcome.html

```diff
@@ -50,7 +50,7 @@
 <div class="container body-content">
     <h1 class="mt-5">Welcome</h1>
     <p>Learn more about <a
-        href="https://github.com/microsoft/spring-cloud-azure/blob/master/azure-spring-cloud-feature-management/README.md">Feature
+        href="https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/feature-management-web-sample/README.md">Feature
```

Comment:
Should this point to the same branch?

Reply:
Should be good enough, will be in master soon. The doc is still accurate.